<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisEditProfilesDialog</class>
 <widget class="QWidget" name="KisEditProfilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>682</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="KPushButton" name="renameButton">
     <property name="text">
      <string>Rename</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QListView" name="profileList"/>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="KPushButton" name="duplicateButton">
     <property name="text">
      <string>Duplicate</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="KPushButton" name="resetButton">
     <property name="text">
      <string>Reset All</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
